contracts/Swap/interfaces/IPancakeSwapV2Router.sol

```diff
@@ -93,7 +93,7 @@ interface IPancakeSwapV2Router {
         uint256 deadline
     ) external payable;
 
-    function swapBnbAndSupplyAtSupportingFee(
+    function swapBNBAndSupplyAtSupportingFee(
```

Comment:
To follow the new naming convention I'd suggest:
```suggestion
    function swapExactBNBForTokensAndSupplyAtSupportingFee(
```

Reply:
Fixed.

Reply:
My bad, pushed it now. :-)